imports/ui/components/root-app-bar.jsx

```diff
@@ -1,39 +1,110 @@
-import React from 'react'
+import React, { Component } from 'react'
 import PropTypes from 'prop-types'
-
 import AppBar from 'material-ui/AppBar'
 import IconButton from 'material-ui/IconButton'
 import FontIcon from 'material-ui/FontIcon'
 import { UneeTIcon } from '../components/unee-t-icons'
-
+import TextField from 'material-ui/TextField'
+import {
+  textInputFloatingLabelStyle,
+  textInputUnderlineFocusStyle,
+  whiteInput
+} from '../components/form-controls.mui-styles'
 import {
   titleStyle,
   logoIconStyle,
   logoButtonStyle
 } from '../components/app-bar.mui-styles'
 
-const RootAppBar = ({ title, onIconClick, shadowless }) => (
-  <AppBar title={title} titleStyle={titleStyle} style={shadowless ? {boxShadow: 'none'} : undefined}
-    iconElementLeft={
-      <IconButton iconStyle={logoIconStyle} style={logoButtonStyle} onClick={onIconClick}>
-        <UneeTIcon />
-      </IconButton>
+class SearchBar extends Component {
```

Comment:
My previous review's comment implied that this component should be taken apart completely, not just moved to its parent component's module. It only contains a TextField and binds its "value" to a prop, so it has no real value of its own: it's just boilerplate code around a TextField. This is why you should move the TextField into the RootAppBar component, and get rid of "SearchBar" completely.